README.md

```diff
@@ -43,9 +44,13 @@ docker-compose -f docker-compose-zipkin-example.yml up
 
 the sample app will run at http://localhost:8081. You should request the URL a few times to generate some sample trace requests!
 
-## Deploying with Kubernetes
+## Deploy in production with Kubernetes
 
-Please refer to the [deployments](https://docs.hypertrace.org/deployments/) section in our documentation which lists the steps to deploy Hypertrace on different Kubernetes flavors across different operating systems and cloud providers. You can find the Helm Charts and installation scripts with more details [here](/kubernetes).
+We support helm charts to simplify deploying Hypertrace in Kubernetes environment, maybe on your on-premise server or cloud instance! 
+
+Please refer to the [deployments section](https://docs.hypertrace.org/deployments/) in our documentation which lists the steps to deploy Hypertrace on different Kubernetes flavors across different operating systems and cloud providers. You can find the Helm Charts and installation scripts with more details [here](https://github.com/hypertrace/hypertrace/tree/main/kubernetes).
+
+`Note:` We have created `hypertrace-ingester` and `hypertrace-service` to simplify local deployment and quick-start with Hypertrace. As of now, we don't support them for production because of some limitations and some unreliabiliy with scaling. So, we will encourage you to deploy individual components for staging as well as production deployments. 
```

Comment:
add a tracking issue for this? Also, it seems odd to say "some limitations and some unreliabiliy with scaling" that sounds a bit more like FUD than anything concrete. Maybe in the tracking issue say what are concretely issues around this as some scaling related details aren't proven with or without a consolidated image

Reply:
the tracking issue can be added as a note here after. For example, "subscribe to [issue NN](link) for updates"